### ISSUE TITLE
hLineColor = white breaks with fillColor

### ISSUE THREAD

**Issue by Christian24** [bpampuch/pdfmake#1192]
Hello,

I have trouble with horizontal table borders that are supposed to be white. It works great for vertical borders, but fails horizontally when there is a cell with a fillColor defined:
<img width="1336" alt="bildschirmfoto 2017-11-18 um 23 07 43" src="https://user-images.githubusercontent.com/2406635/32985229-2d91a46c-ccb6-11e7-9d83-fbd53bea3c61.png">
Notice the weird top border above Projekte Anlage 1. The white border is supposed to be like it is, but then it seems to fade back into the fillColor of the table. (The black rectangle is just to protect potentially sensible information. ) I also have that effect on bigger tables with rows having a fillColor so it is not limited to a single cell. Interestingly I do not notice it for vertical borders and it is gone if I change the color to anything but white. This is my code to generate the table: 

````typescript
 const table: any = {
    style: 'risikoTable',
    table: {
      headerRows: 1,
      widths: [50, 100, 50, 50, 50, 50, 80],
      body: [
        [{text: overview.groupName, fillColor: '#1D3652', color: 'white', colSpan: 2}, {}, {
          text: 'Qualitäten', bold: true,
          color: '#1D3652'
        }, {
          text: 'Termine', bold: true,
          color: '#1D3652'
        }, {
          text: 'Kosten', bold: true,
          color: '#1D3652'
        }, {
          text: 'Chancen', bold: true,
          color: '#1D3652'
        },
          {
            text: 'Gesamtprognose', bold: true,
            color: '#1D3652'
          }]
      ]
    },
    layout: {
      fillColor: function (i, node) {
        if (i === 0) {
          return null;
        } else {
          return (i % 2 === 0) ? '#F7F6FB' : null;
        }

      },
      hLineColor: function (i, node) {
        return 'white';
      },
      vLineColor: function (i, node) {
        return 'white';
      },
      hLineWidth: function (i, node) {
        return 1;
      },
      vLineWidth: function (i, node) {
        return 1;
      },

    }
  };
````

Thanks a lot in advance.

**Comment by dorothythurston** [comment]
Hi! Can you update your code sample so it can be exactly copied into the playground so others can help look into it? Right now you have some variables that are missing. I have found the best way to get others to help you find out the problem is to submit a code sample that when pasted runs in the playground. You will need to make sure you don't have any variables missing and add the necessary outside pieces like: `var dd = {
	content: [
	]	
}`

**Comment by liborm85** [comment]
Same as https://github.com/bpampuch/pdfmake/issues/1609. Fixed in 0.1.50.
